container/secrets.go

```diff
@@ -81,5 +86,8 @@ func readFile(root, name string) ([]SecretData, error) {
 }
 
 func getHostSecretData() ([]SecretData, error) {
-	return readAll("/usr/share/rhel/secrets", "")
+	if _, err := os.Stat(overrideDir); err == nil {
```

Comment:
alright so it's a complete merge overriding from `/etc/container/` and in case of an empty dir in `/etc/container/` we grab everything from `/usr/share/rhel/secrets/`